ProcessMaker/processmaker
# Fix Issue Causing Screen Title Overwriting When Project is Selected

This PR resolves the issue encountered when creating a screen, where the screen title gets removed when a project is selected. The solution includes incorporating the 'projects' property within the formData object to prevent the title from being overwritten when a project is added.

## Solution
- Added the 'projects' property to the formData object to maintain the screen title when a project is selected.

## How to Test

1. Ensure that a project is properly configured within the system.
2. Navigate to Designer > Screen and proceed to create a new screen.
3. Fill out the necessary form details.
4. While selecting a Project, ensure that the screen name remains unchanged and is not overwritten by the project selection.

## Related Tickets & Packages
- [FOUR-11221](https://processmaker.atlassian.net/browse/FOUR-11221)
ci:next

## Code Review Checklist


[FOUR-11221]: https://processmaker.atlassian.net/browse/FOUR-11221?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
